### PR TITLE
Use Database type instead of string 

### PR DESCRIPTION
**Pull request by umangbp**
Signed-off-by: Umang Patel <umang.patel@healthengine.com.au>

### Description

This PR refactors tryColumnCreationQuery function and all its occurrences to use `DatabaseName` type instead of `string` type for db name

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]

**Comment by umangbp** [comment]
@williamdes @MauricioFauth Hey Guys, I am not sure why end to end tests are failing. It does not look like anything related to the changes I made is affecting it. Happy to receive any feedback if it's my changes failing that.

**Comment by umangbp** [comment]
@williamdes can you please assign this issue to me.

File: libraries/classes/Controllers/Export/ExportController.php [hunk 1]
```diff
@@ -409,10 +410,10 @@ public function __invoke(ServerRequest $request): void
                 }
 
                 if (isset($GLOBALS['lock_tables'])) {
-                    $this->export->lockTables($GLOBALS['db'], $GLOBALS['tables'], 'READ');
+                    $this->export->lockTables(DatabaseName::fromValue($GLOBALS['db']), $GLOBALS['tables'], 'READ');
                     try {
                         $this->export->exportDatabase(
-                            $GLOBALS['db'],
+                            DatabaseName::fromValue($GLOBALS['db']),
```
Comment (MauricioFauth) [inline libraries/classes/Controllers/Export/ExportController.php:416]: You are creating a new object each time, you could assign the database name to a variable. For example:

```php
$db = DatabaseName::fromValue($GLOBALS['db']);
```

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Review by MauricioFauth — APPROVED** [review]

**Comment by MauricioFauth** [comment]
Merged, thanks for your contribution!

**Review by williamdes — COMMENTED** [review]
👍🏻

**Review by williamdes — COMMENTED** [review]
👁️ (sorry GH still review requests me)

**Pull request merged**